apps/mobile/mobile-app/components/impact/impact-stat.tsx

```
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Pressable, View } from 'react-native'
+import { Text } from '../Themed'
+import { Box } from '../ui/box'
+import { Icon } from '../ui/icon'
+
+type ImpactStatProps = {
+	icon: any // Icon component from lucide-react-native
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)